packages/seacas/libraries/ioss/src/CMakeLists.txt

```diff
@@ -43,8 +43,7 @@ IF (NOT ${PROJECT_NAME}_ENABLE_Zoltan)
   ADD_DEFINITIONS(-DNO_ZOLTAN_SUPPORT)
 ENDIF()
 
-IF (${PROJECT_NAME}_ENABLE_ParMETIS)
-ELSE()
+IF (NOT TPL_ENABLE_ParMETIS)
```

Comment:
So you decided not to change this to `${PACKAGE_NAME}_ENABLE_ParMETIS`?

Reply:
No, it wasn't working so I went back to `TPL_ENABLE_ParMETIS` until I could figure out why the `${PACKAGE_NAME}` version wasn't working...

Reply:
> No, it wasn't working so I went back to `TPL_ENABLE_ParMETIS` until I could figure out why the `${PACKAGE_NAME}` version wasn't working...

Okay, we can look into this later to see what is up with that.